duckduckgo/Android
# https upgrade

Asana Issue URL: https://app.asana.com/0/488551667048375/364367284740780

## Description

Upgrades URLs to https

## Steps to Test this PR:

1. Visit http://english.sogou.com
1. Observe address becomes https
